mono/metadata/w32handle.c

```diff
@@ -343,7 +384,7 @@ static guint32 mono_w32handle_new_internal (MonoW32HandleType type,
 					  gpointer handle_specific)
 {
 	guint32 i, k, count;
-	static guint32 last = 0;
+	guint32 last = 0;
```

Comment:
@luhenry yeah it is a silly weird change and should not be merged. This is just to exasperate reuse of empty slots in the hope of making the w32handle bug more frequent. It is not an important change and I should be benign aside from making allocation slower.